README.md

```diff
@@ -74,6 +75,8 @@ Rules are grouped by category to help you understand their purpose.
 
 All rules below with a check mark :white_check_mark: are enabled by default while using `plugin:ember/recommended` config.
 
+All rules below with a red car :car: are enabled by default while using `plugin:ember/octane` config.
```

Comment:
This line also needs to make it clear that all recommended rules are included in this config too. Perhaps like this:

```
The `plugin:ember/octane` config contains both Octane rules with a red car :car: in addition to the rules in the `plugin:ember/recommended` config.
```

Reply:
@bmish updated! If you want me to squash before you merge let me know.